inputs/LiMetal_PorousSep_oxygen.yaml

```diff
@@ -49,6 +49,11 @@ cell-description:
     Minrad: 1e-10 #m
     Maxrad: 1e-6 #m
     buckets: 200 #number
+    stored-species: # Details on the stored species. Used for capacity calc.
+      name: Li[metal]
```

Comment:
Should be `Li2O2`, no?

Reply:
yes, do I need quotes on this like the other species?

Reply:
Yes, I think so.